### ISSUE TITLE
Elasticsearch can't start

### ISSUE THREAD

**Issue by mhmdio** [cn-terraform/terraform-aws-sonarqube#5]
Fixing for the error, Elasticsearch can't start
```

[1]: max file descriptors [4096] for elasticsearch process is too low, increase to at least [65535]
--
 
```

update ecs fargate module by adding the following vars:
```hcl
  command = [
    "-Dsonar.search.javaAdditionalOpts=-Dnode.store.allow_mmapfs=false"
  ]
  ulimits = [
    {
      "name" : "nofile",
      "softLimit" : 65535,
      "hardLimit" : 65535
    }
  ]
```

**Comment by jnonino** [comment]
Hi, how are you?
Have you tested that? If that works for you, please submit a PR and I'll review it and generate a new version of the module with your changes!!

**Comment by mhmdio** [comment]
Hey @jnonino, Yes it worked for me, I created two issues that I Found, both suggested fixes worked. will submit PRs. Also, I used RDS without Aurora to keep the price low, is this something we can submit too?

**Comment by jnonino** [comment]
Hi @mhmdio. Sure, any improvement or suggestion is more than welcome.

**Comment by jnonino** [comment]
Fix released in version [2.0.14](https://registry.terraform.io/modules/cn-terraform/sonarqube/aws/2.0.14)
